guides/common/modules/proc_attaching-red-hat-subscriptions-to-content-hosts.adoc

```diff
@@ -10,7 +10,7 @@ For more information about activation keys, see xref:Managing_Activation_Keys_{c
 
 .Smart Management Subscriptions
 
-In {Project}, you must maintain a Red{nbsp}Hat Enterprise Linux Smart Management subscription for every Red{nbsp}Hat Enterprise Linux host that you want to manage.
+In {Project}, you must maintain a {RHEL} Smart Management subscription for every {RHEL} host that you want to manage.
```

Comment:
Honestly, I didn't look too carefully. The whole chapter is about attaching Red Hat Subscriptions. And `{RHEL}` should (IMHO) never expand to `AlmaLinux`.

It may be Satellite specific, but let's keep the scope to just replacing `Red Hat Enterprise Linux` with `{RHEL}` (where appropriate).